### PR TITLE
Major Changes to add Global Configuration Management (GCM), adding LHR/FRA, and Monitoring Tools

### PR DESCRIPTION
**Pull request by nhmartinez**
Questions that were updated with GCM, Global Sign-in, Lex Global Resiliency and CodeCommit no longer accepting new customers:   
     •   Under what conditions would you recover back to the initial source AWS region?
     •   How are you ensuring your other AWS services are configured to meet your multi-region strategy?
     •   How are you ensuring configuration parity between your two Amazon Connect instances?
     •   How do you dynamically adapt to out-of-office conditions?
     •   ACGR will mirror Amazon Connect resources, what other resources are you mirroring between your two regions?
     •   How are you accounting for your metrics and data between your two instances?
     •   What is your agent failover mechanism?
     •   How have you configured your Traffic Distribution Groups (TDG)?
     •   What multi region strategy are you considering?
     •   Are you checking whether agents have completed their active calls before failing them over?
     •   What resiliency considerations for Amazon Lex have you implemented?
     •   What resiliency considerations for AWS Lambda have you implemented?
     •   What configuration options to enable Amazon Connect Global Resiliency do you have in place today?
     •   What does your failover runbook address?

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lenses/connect-customlens-globalResiliency.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
 						},
 						{
 							"id": "choice2",
-							"title": "Created a test SAML 2.0-enabled Amazon Connect instance in regions where multi-region is available (us-east-1/ us-west-2)",
+							"title": "Created a test/dev SAML 2.0-enabled Amazon Connect instance in regions where ACGR is available (IAD/PDX or LHR/FRA)",
```
Comment (sfred) [inline lenses/connect-customlens-globalResiliency.json]: We don't define regions like this externally, so we should revert back to terms like us-east-1

File: lenses/connect-customlens-globalResiliency.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
 						},
 						{
 							"id": "choice3",
-							"title": "Created a production SAML 2.0-enabled Amazon Connect instance in regions where multi-region is available (us-east-1/ us-west-2)",
+							"title": "Created a production SAML 2.0-enabled Amazon Connect instance in regions where ACGR is available (IAD/PDX or LHR/FRA)",
```
Comment (sfred) [inline lenses/connect-customlens-globalResiliency.json]: Same here

File: lenses/connect-customlens-globalResiliency.json [hunk 3]
```diff
@@ -210,41 +210,41 @@
...
 							"improvementPlan": {
-								"displayText": "Identify how you would like to distribute your traffic across your business.",
+								"displayText": "Identify how you would like to distribute your traffic and agents across your business.",
 								"url": "https://docs.aws.amazon.com/connect/latest/adminguide/setup-traffic-distribution-groups.html"
 							}
 						},
 						{
 							"id": "choice3",
-							"title": "All production numbers are in one TDG and all test numbers are in another TDG.",
+							"title": "All production numbers and agents are in one TDG and all test numbers and agetns are in another TDG.",
```
Comment (sfred) [inline lenses/connect-customlens-globalResiliency.json]: Spelling

File: lenses/connect-customlens-globalResiliency.json [hunk 4]
```diff
@@ -331,8 +344,8 @@
 							}
 						},
 						{
-							"id": "choice5",
-							"title": "Why and how you move a particular telephony distribution group before another",
+							"id": "choice6",
+							"title": "Why and how you move a particular traffi distribution group before another",
```
Comment (sfred) [inline lenses/connect-customlens-globalResiliency.json]: Spelling

File: lenses/connect-customlens-globalResiliency.json [hunk 5]
```diff
@@ -544,7 +569,7 @@
 				{
 					"id": "GlobalResiliency_09",
 					"title": "Are you checking whether agents have completed their active calls before failing them over?",
-					"description": "Shifting agent logins can be disruptive if they are actively participating in an interaction.",
+					"description": "Shifting agent logins can be disruptive if they are actively participating in a non-voice interaction.",
```
Comment (sfred) [inline lenses/connect-customlens-globalResiliency.json:572]: Can you clarify this? The question implies it's about voice interactions, while the answers are talking about non-voice interactions.

File: lenses/connect-customlens-globalResiliency.json [hunk 6]
```diff
@@ -796,23 +737,35 @@
...
+								"displayText": "ACGR instances onboarded from Nov-2023 have Global Configufation Management enabled by default, previous customers will need to follow a procedure to enable it.",
+								"url": "https://docs.aws.amazon.com/connect/latest/adminguide/get-started-connect-global-resiliency.html"
+							}
+						},						
+						{
+							"id": "choice2",
 							"title": "Manual Backup and Restore.",
 							"helpfulResource": {
-								"displayText": "Amazon Connect and other AWS services provide manual back and restore functionality for migrating configurations across environments.",
+								"displayText": "If ACGR Global Configuration Management is not enabled (pre Nov-2023), Amazon Connect and other AWS services provide manual back and restore functionality for migrating configurations across environments.",
```
Comment (sfred) [inline lenses/connect-customlens-globalResiliency.json:759]: Where is this pre-Nov 2023 date communicated publicly? If this isn't documented, we shouldn't include it as these lenses are not designed to be the source of truth

**Review by sfred — CHANGES_REQUESTED** [review]
Left a couple comments @nhmartinez

File: lenses/connect-customlens-globalResiliency.json (hunk 1 above)
Comment (nhmartinez) [reply lenses/connect-customlens-globalResiliency.json]: Updated in the new commit

File: lenses/connect-customlens-globalResiliency.json (hunk 2 above)
Comment (nhmartinez) [reply lenses/connect-customlens-globalResiliency.json]: Updated in the new commit

File: lenses/connect-customlens-globalResiliency.json (hunk 3 above)
Comment (nhmartinez) [reply lenses/connect-customlens-globalResiliency.json]: Updated in the new commit

File: lenses/connect-customlens-globalResiliency.json (hunk 4 above)
Comment (nhmartinez) [reply lenses/connect-customlens-globalResiliency.json]: Updated in the new commit

File: lenses/connect-customlens-globalResiliency.json (hunk 5 above)
Comment (nhmartinez) [reply lenses/connect-customlens-globalResiliency.json:572]: Updated to Interaction in the new commit, with Global Sign-in the agent will not be moved until the Voice call ends, that does not apply for non-voice

File: lenses/connect-customlens-globalResiliency.json (hunk 6 above)
Comment (nhmartinez) [reply lenses/connect-customlens-globalResiliency.json:759]: Yes, it does, it is in our Release Notes

**Review by sfred — APPROVED** [review]
Looks good!

**Pull request merged**